src/setup-ros-windows.ts

```diff
@@ -4,17 +4,54 @@ import * as chocolatey from "./package_manager/chocolatey";
 import * as pip from "./package_manager/pip";
 import * as utils from "./utils";
 
-const rosdepBin: string =
-	"c:\\hostedtoolcache\\windows\\python\\3.6.8\\x64\\scripts\\rosdep";
+const python37: string =
+	"C:\\hostedtoolcache\\windows\\Python\\3.7.6\\x64";
+
+const binaryReleases: { [index: string]: string; } =
+	{
+		"dashing": "https://github.com/ros2/ros2/releases/download/release-dashing-20191213/ros2-dashing-20191213-windows-amd64.zip",
+		"eloquent": "https://github.com/ros2/ros2/releases/download/release-eloquent-20200124/ros2-eloquent-20200124-windows-release-amd64.zip"
+	};
+
+const pip3Packages: string[] = [
+	"numpy",
+	"netifaces",
+	"lxml"
+];
 
 /**
- * Install ROS 2 on a Windows worker.
+ * Install ROS 2 build tools.
  */
-export async function runWindows() {
+async function prepareRos2BuildEnvironment() {
+	await utils.exec(`cmd /c mklink /d c:\\python37 ${python37}`);
+	core.exportVariable("PYTHONHOME", "c:\\Python37");
```

Comment:
Case sensitive? The line above has a lowercase `p`

Reply:
I think this would work since Windows is not case sensitive, however I would like it to be consistent

Reply:
I tried to make all lower cases for consistency. Let me know if you find anything I missed. :p